scripts/staking/output/staking_snapshot_RSK_block_8098932_2025-10-14T10-37-37-000Z.json

```diff
@@ -0,0 +1,1787 @@
+        "votingPower": "24342.094238248003912752",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x20942a70efb66c9280afee609fb214651765124d",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x20f6322bf83cb8b86439c260bc82562d8d567f97",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x210781c0a8a192b9edf20d0a3e297c1789256779",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x2153c971854436938031aa105f28558011f4518f",
+        "amountStaked": "2663.484810589303831733",
+        "votingPower": "20242.48456047870912117",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x225b292e2523103b546d83e3ae58c6e57527e2a8",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x2340d8265a6108e58e4d90a84d3e6df3a3d61319",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x26fb82cd87338268a77825f168e619d6502e2c99",
+        "amountStaked": "20000.237100610510460235",
+        "votingPower": "28000.331940854714644329",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x27afca928750df50e68185d576f678db351ff3a4",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x283a49b59846dce46752b7ebb7ba5df7a8564960",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x287a3bcb53c90f190aed3397d9aeca629766ec9a",
+        "amountStaked": "306155.561426232099646839",
+        "votingPower": "2782147.807770143314568343",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x2b1e65441cbbcfb1b91d987e63642911991f2afa",
+        "amountStaked": "40.0",
+        "votingPower": "0.0",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x2b45400a61927e8ba22e379ea00498dc573329c6",
+        "amountStaked": "10000.000076923076923178",
+        "votingPower": "77000.00059230769230847",
+        "votingPowerMode": "voluntary",
+        "isVesting": false
+    },
+    {
+        "address": "0x2c85892e5188d47019492ad0711cffaa6d777f01",
+        "amountStaked": "0.0",
+        "votingPower": "0.0",
```

Comment:
why are there 0 amountsStaked in the list?  
pls make sure that you get correct addresses - should be only those provided as staking addresses by origins users.